src/Exception/OperationTimedOut.php

```diff
@@ -13,4 +13,20 @@
 
 class OperationTimedOut extends \Exception
 {
+    public function __construct(int $timeoutMicroSec, int $code = 0, ?\Throwable $previous = null)
```

Comment:
Increase of replacing the constructor, could you add a `public static function createFromTimeout(...)`?

Reply:
Done.